plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -318,10 +328,10 @@ public Optional<AggregationApplicationResult<ConnectorTableHandle>> applyAggrega
                 .flatMap(List::stream)
                 .distinct()
                 .peek(handle.getColumns().<Consumer<JdbcColumnHandle>>map(
-                                columns -> groupKey -> verify(columns.contains(groupKey),
-                                        "applyAggregation called with a grouping column %s which was not included in the table columns: %s",
-                                        groupKey,
-                                        tableColumns))
+                        columns -> groupKey -> verify(columns.contains(groupKey),
+                                "applyAggregation called with a grouping column %s which was not included in the table columns: %s",
+                                groupKey,
+                                tableColumns))
```

Comment:
unrelated change, separate commit

Reply:
Reverted. The formatter changed its mind.